resource/templates/extract-recipe-template.html

```diff
@@ -48,18 +48,18 @@ <h2 class="settings-section-title">GPU</h2>
                <label class="control-label">GPU Memory</label>
                <div class="controls">
                   <select dku-bs-select
-                     name="gpu_memory"
-                     ng-model="config.gpu_memory"
-                     ng-options="gm.value as gm.label for gm in gpuMemory">
+                     name="gpu_memory_allocation_mode"
```

Comment:
There is some duplication of code here (same as for JS files). Could you have a dedicated template for GPU-related stuff and call that template from each recipe template?

Reply:
Fixed in c06d932